node/_fs/_fs_util.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
It looks like the methods added in this file are already implemented in [this file](https://github.com/denoland/deno_std/blob/main/node/internal/fs/utils.mjs).

Reply:
:+1: fixed https://github.com/denoland/deno_std/pull/2692/commits/e19d20f2f7d5e0cfc00fb3ae74021f5686f3f517